cmd/get/appcatalogs/flag.go

```diff
@@ -0,0 +1,34 @@
+package appcatalogs
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const (
+	flagLabelSelector     = "selector"
+	publicCatalogSelector = "application.giantswarm.io/catalog-visibility=public"
```

Comment:
Command defaults to show the public catalogs but it is possible to override this via the flag.